query/src/main/scala/filodb/query/exec/rangefn/RangeFunction.scala

```diff
@@ -436,11 +441,11 @@ extends ChunkedRangeFunction[R] {
 
     // update timestamp only if
     //   1) endRowNum >= 0 (timestamp within chunk)
-    //   2) timestamp is within stale window; AND
+    //   2) timestamp is within window; AND
     //   3) timestamp is greater than current timestamp (for multiple chunk scenarios)
     if (endRowNum >= 0) {
       val ts = tsReader(tsVectorAcc, tsVector, endRowNum)
-      if ((endTime - ts) <= queryConfig.staleSampleAfterMs && ts > timestamp)
```

Comment:
Why are we removing staleSampleAfterMs?

Reply:
@TanviBhavsar this is to make sure that lookback time falls within the `window`, right now even if the window is 3mins, lookback time can go upto 5mins because of `queryConfig.staleSampleAfterMs`.

Reply:
@sherali42 the above check is still needed. If window was given as 30mins, the last sample holds good only for 5 mins. That check needs to stay.

Reply:
ok Sher convinced me that since data worth only a 5 minute (stale window length) lookback window will be scanned, we should be fine removign the condition.